Src/Fido2/Fido2.cs

```diff
@@ -62,21 +62,14 @@ public CredentialCreateOptions RequestNewCredential(
     /// <summary>
     /// Verifies the response from the browser/authenticator after creating new credentials.
     /// </summary>
-    /// <param name="attestationResponse">The attestation response from the authenticator.</param>
-    /// <param name="originalOptions">The original options that was sent to the client.</param>
-    /// <param name="isCredentialIdUniqueToUser">The delegate used to validate that the CredentialID is unique to this user.</param>
-    /// <param name="requestTokenBindingId">DO NOT USE - Deprecated, but kept in code due to conformance testing tool</param>
+    /// <param name="makeNewCredentialParams">Wraps the input parameters for making a new credential</param>
```

Comment:
Description can be simplified to: 

-> The input parameters for making a new credential